app/components/ui/molecules/DashboardListItem.md

```diff
@@ -1,11 +1,44 @@
-List item in the dashboard that links to manuscript. It shows status (e.g. accepted, waiting approval), manuscript title and date since status last changed.
+List item in the dashboard. It shows the manuscript title, submission status & the date of the last status change.
 
 ```js
+const today = new Date()
 ;<DashboardListItem
   statusCode={'CONTINUE_SUBMISION'}
   title={
     'Shearing in flow environment promotes evolution of social behaviour in microbial populations'
   }
+  date={today}
+/>
+```
+
```

Comment:
The list items don't appear on the styleguide other than the first one. 

Take a look at the [FileUpload.md](https://github.com/elifesciences/elife-xpub/blob/develop/app/components/ui/molecules/FileUpload.md) for the correct usage.

Reply:
I'm surprised that not having text in between code blocks would make a difference.

For the file upload, they are distinctly different states & therefore easily labelled as something different. I'm not sure what text I would put at the top of these, other than just re-iterating what you see below ('Different title, status = Submitted, today's date ?) :woman_shrugging:

Reply:
Now that the styleguide PRs have been merged, seems to be fine to me:
![image](https://user-images.githubusercontent.com/15656538/47584591-d9d59900-d952-11e8-9637-9d0a120ad921.png)